src/geo/torque-layer-view-base.js

```diff
@@ -0,0 +1,160 @@
+var _ = require('underscore');
+
+/**
+ * Implementation of all common logic of a torque-layer view.
+ * All the methods and attributes here need to be consistent across all implementing models.
+ *
+ * Methods are prefixed with _ to indicate that they are not intended to be used outside the implementing models.
+ */
+module.exports = {
+
+  _initialAttrs: function (layerModel) {
+    var extra = layerModel.get('extra_params');
+    return {
+      table: layerModel.get('table_name'),
+      user: layerModel.get('user_name'),
+      column: layerModel.get('property'),
+      blendmode: layerModel.get('torque-blend-mode'),
+      resolution: 1,
+      // TODO: manage time columns
+      countby: 'count(cartodb_id)',
```

Comment:
sql_api_*, tiler_*, animationDuration, countby, steps, blendmode, column and resolution could be removed safely (since we don't need backwards compatibility).

- sql_api_* is not being used
- tiler_* was replaced by maps_api_template
- the other ones go in the cartocss